my-dsl.md

```diff
@@ -1,38 +1,59 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+_What is the name of the language? Link the name to its webpage_
 
+[Erlang](https://www.erlang.org/)
 
 # Domain
 _Describe the language's domain in five words._
 
+Massively scalable real-time systems
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+The code is first compiled.  Then it gets run on a C emulator.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+I think it classifies as a domain specific langauge, but I think it is more on
+the general purpose side of the spectrum because it has control structures like
+loops which make it have a less limited expressiveness.  However it is still
+clearly targeted at specific domains where massive real time scalability is
+required.  It has a sense of fluency in the way expressions come together
```

Comment:
Makes sense, even if the language could be used for more general purpose, it is clearly targeted toward systems uses.